src/main/java/com/google/sps/data/DataManager.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
```

Comment:
Overall, just to keep style consistent with the other Java files, I recommend do these:
- Use 4 spaces for indentation rather than 2 spaces (you might have to replace some tabs with 4 spaces because this gets messed up on Github, also pressing tab should automatically put in 4 spaces currently). Also fix the places where the indentation is inconsistent.
- When writing loops/functions/methods, put the opening bracket on the same line as the first part of the loop/function/method. Also put a space before the opening bracket. Example:
method blah {

}